packages/service-library/src/servicelib/utils_secrets.py

```diff
@@ -68,7 +68,7 @@ def _is_possibly_sensitive(name: str, sensitive_keywords: set[str]) -> bool:
 
 
 def mask_sensitive_data(
-    data: dict, *, extra_sensitive_keywords: set[str] | None = None
+    data: dict[str, Any], *, extra_sensitive_keywords: set[str] | None = None
```

Comment:
Actually wanted to ask about the use of this function with regards to the e2e tests.
The point was actually to be able to copy/paste from the e2e logs directly the code to re-run locally. is it still possible with the use of this?

Reply:
@sanderegg lets check this offline. It risks of ending up in a "spaces vs tabs" discussion :-)